workflow/scripts/extract-fp-fn.py

```
@@ -28,6 +29,44 @@
 
+def _get_vaf_from_record(record, field, name):
+    """Extract VAF value from a record, handling FORMAT and INFO fields.
+
+    Returns a scalar float VAF (first value if array-like), or nan if not found.
+    """
+    try:
+        if field == "INFO":
+            vaf = record.info.get(name)
+        else:
+            # FORMAT field - use samples dict
+            sample_name = list(record.samples.keys())[0]
```

Ruff (0.15.20)

[warning] Prefer `next(iter(record.samples.keys()))` over single element slice

Replace with `next(iter(record.samples.keys()))`

(RUF015)